microservices/provision/templates/provisioning/YealinkT21P_E2/specific.cfg

```diff
@@ -1,11 +1,11 @@
 #!version:1.0.0.1
-account.1.user_name = <?php echo $this->terminal->getName(); ?> 
-account.1.auth_name = <?php echo $this->terminal->getName(); ?> 
-account.1.password = <?php echo $this->terminal->getPassword(); ?> 
-account.1.display_name = <?php echo $this->user->getName(); ?> 
-account.1.label = <?php echo $this->user->getName(); ?> 
-account.1.sip_server_host = <?php echo $this->company->getDomainUsers(); ?> 
-account.1.sip_server_port = 5060 
+account.1.user_name = <?php echo $this->terminal->getName() . "\n" ; ?>
+account.1.auth_name = <?php echo $this->terminal->getName() . "\n"; ?>
+account.1.password = <?php echo $this->terminal->getPassword() . "\n"; ?>
+account.1.display_name = <?php echo $this->user->getName() . "\n"; ?>
+account.1.label = <?php echo $this->user->getName() . "\n"; ?>
+account.1.sip_server_host = <?php echo $this->company->getDomainUsers() . "\n"; ?>
+account.1.sip_server_port = 5060
```

Comment:
The original templates seem to have a \n already at the end of the line and terminals worked alright, this will send two \n, one implicit and another at the end of the line. I assume there will be no problem for the terminals, but extra spacing should be not need.

Reply:
new lines after php closing tag (?>) are lost at php parsing: https://stackoverflow.com/questions/1908175/is-there-a-way-to-force-a-new-line-after-php-closing-tag-when-embedded-among

That's why we used to have an empty space at the end of the lines but that spaces conflict with "remove trailing spaces" configuration on most IDEs and it was causing me issues on tests